.github/workflows/update-baselines.yml

```diff
@@ -0,0 +1,36 @@
+name: Update Baselines
+
+on:
+  workflow_dispatch:
+
+jobs:
+  main:
+    name: Update Baselines
+    runs-on: ubuntu-latest
+
+    env:
+      # don't print dotnet logo
+      DOTNET_NOLOGO: true
+
+      # disable telemetry (reduces dotnet tool output in logs)
+      DOTNET_CLI_TELEMETRY_OPTOUT: true
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v1.7.2
+      
+      - name: Test
+        continue-on-error: true
+        run: dotnet test -- 'TestRunParameters.Parameter(name="SetBaseLine", value="true")'
+      
+      - name: Commit baselines
```

Comment:
Where does it commit to? And can you add a comment with some instructions how to use it?

Reply:
![image](https://user-images.githubusercontent.com/38542602/109993040-e2d09500-7cd9-11eb-8f6a-dcd737d63155.png)
It's a workflow action that you can run on-demand on a branch (example of the equivalent on the `Build` action). The commit is made to the branch you run it on. I'll add this info to the action.

Reply:
Oh I forgot that the branch is automatically added as a parameter on these. Nice!